app/assets/stylesheets/mobile/header.css.scss

```diff
@@ -169,10 +163,20 @@ header {
 }
 
 /* This class is added when the user open the drawer */
-#app.draw > * {
-          transform: translateX(-80%);
-       -o-transform: translateX(-80%);
-      -ms-transform: translateX(-80%);
-     -moz-transform: translateX(-80%);
-  -webkit-transform: translateX(-80%);
+.draw {
```

Comment:
I do not have an Ipad unfortunately. I tested in on Firefox Desktop, Chrome for Android on 2 phones and a tablet, and Firefox for Android. I will add in a few moment Opera Mobile to test it there. But I'm afraid that would not help, as Opera Mobile doesn't support transforms, according to caniuse.com